File: client/src/features/services/dto/response/service.response.ts
import type { UserBriefResponse } from "../../../../shared/dto/response";

export interface ServiceResponse {
  id: string;
  productId: string;
  productName: string;
  name: string;
  status: "ACTIVE" | "INACTIVE";
  createdBy: UserBriefResponse;
  updatedBy: UserBriefResponse;
  createdAt: string;
  updatedAt: string;
}

File: backend/sentinel-api/src/main/java/com/sentinel/api/service/dto/response/ServiceResponse.java
package com.sentinel.api.service.dto.response;

import com.sentinel.server.common.dto.response.UserBriefResponse;
import com.sentinel.server.service.entity.ServiceStatus;
import java.time.Instant;

public record ServiceResponse(
        String id,
        String productId,
        String productName,
        String name,
        ServiceStatus status,
        UserBriefResponse createdBy,
        UserBriefResponse updatedBy,
        Instant createdAt,
        Instant updatedAt) {
}
